### DISCUSSION TITLE
statusCode 500 & upnp ErrorCode 501

### DISCUSSION THREAD

**Discussion by MarioSpanicciati** [hklages/node-red-contrib-sonos-plus#134] [Q&A]
Hi, I keep receiving the following statusCode 500 & upnp ErrorCode 501 no matter which command I am executing.  I have been able to discover my device successfully and tried configuring access via Serial Number and separately via IP Address.

Error message: "group.play.tunein: statusCode 500 & upnpErrorCode 501. upnpErrorMessage >>Action Failed :: Details: none"

Flow: 
<img width="837" alt="image" src="https://user-images.githubusercontent.com/80224212/110354098-a4deb400-7fec-11eb-88bf-9f5144ef29fd.png">

**Comment by hklages** [comment]
Hi Mario.

Could you please try the command "household.get.groups" with a debug node
on the output. Does that work? Do you get an array of your player?
It seems that the player is reachable but the TuneIn station can not be
played. What station did you choose?
What Node-RED, NodeJS do you use?

Henning

**Reply by MarioSpanicciati** [reply]
Hi Henning,

Thank you.  I seem to receive this error for anything that I attempt to do.  Here's a screenshot of the debug output.

My versions are:
Node-RED version: v1.2.9
Node.js  version: v14.16.0

<img width="1294" alt="image" src="https://user-images.githubusercontent.com/80224212/110378207-298bfb00-800a-11eb-8e58-f5bc1a020fe4.png">

**Comment by hklages** [comment]
Hmmmm - strange. 

- in your browser could you please just enter the `http://<ip of sonos player>:1400/info` and attach the output here (please replace the householdid with '*' or delete it completly. If you are concerned about data privacy  - just send an email to `nrcsplus@gmail.com`

- send a screen shot of your config node. 

What system do you use? Raspberry Pi, CCU3 ?

Do you have access to the log file - can you send it?
